### ISSUE TITLE
Make it possible to update command configurations without redeploying

### ISSUE THREAD

**Issue by joao-paulo-parity** [paritytech/command-bot#60]
Problem: Because command configurations are stored in [commandConfigurations](https://github.com/paritytech/command-bot/blob/4c7c6413fe2c1f0a47db6f96eac65d6fccd91d71/src/core.ts#L19) we have to redeploy the application in order to change it

Solution: move the configuration to https://gitlab.parity.io/parity/infrastructure/ci_cd/configuration and fetch it from there.

**Comment by TriplEight** [comment]
And between these predefined jobs, we could have some generic ones such as
```
linux-docker-generic-job:
  stage:                           stage4
  rules:
    - if: $ARBITRARY_JOB
  script:
    - cargo $ARBITRARY_JOB
  tags:
    - linux-docker
```

And the bot would just pass the payload to `cargo ...` in a `$ARBITRARY_JOB` variable. This way, CI will only run just that job.
And such jobs templates we'd separate into an `include`d file like https://github.com/paritytech/substrate/blob/master/.gitlab-ci.yml#L172

**Comment by TriplEight** [comment]
And between these predefined jobs, we could have some generic ones such as
```
linux-docker-generic-job:
  stage:                           stage4
  rules:
    - if: $ARBITRARY_JOB
  script:
    - cargo $ARBITRARY_JOB
  tags:
    - linux-docker
```

And the bot would just pass the payload to `cargo ...` in a `$ARBITRARY_JOB` variable. This way, CI will only run just that job.
And such jobs templates we'd separate into an `include`d file like https://github.com/paritytech/substrate/blob/master/.gitlab-ci.yml#L172

**Comment by joao-paulo-parity** [comment]
> My initial idea was that it would be better off if we'd have predefined jobs in the CI config instead of hardcoded into the bot.

It is feasible. I didn't want to have it for the MVP for multiple reasons, one of them being that I didn't want to change the repositories' CI files because of this project as that would be more complicated and take longer. The current approach is more flexible and easier to experiment with.

> And such jobs templates we'd separate into an included file

Job templates would definitely be needed as the job configuration has more than just the command.

https://github.com/paritytech/command-bot/blob/4c7c6413fe2c1f0a47db6f96eac65d6fccd91d71/src/gitlab.ts#L58-L99

Optimally we'd fetch the template from a repository like https://github.com/paritytech/scripts instead of having to duplicate them for each project.

**Comment by mordamax** [comment]
It's already solved with #126
